### ISSUE TITLE
Patch instructions for 3rd party image

### ISSUE THREAD

**Issue by sjezewski** [pachyderm/pachyderm#1060]
Right now we have a mismatch between job-shim on `HEAD` and pachd at `1.2.3` (the latest point release).

This causes the issue where you see:

```
 env key not set when required: PPS_POD_NAME main.appEnv
```

From new job pods.

As a temporary solution, we can replace the `go get` line in the Dockerfile with:

```
 RUN go get github.com/pachyderm/pachyderm
RUN cd  /go/src/github.com/pachyderm/pachyderm && \
    git checkout v1.2.3
RUN cd /go/src/github.com/pachyderm/pachyderm && \
    go install ./src/server/cmd/job-shim && \
    cp $GOPATH/bin/job-shim /job-shim 
```

[When this PR](https://github.com/pachyderm/pachyderm/pull/1041) lands, this issue should go away, but in the meantime we should make sure the instructions work.

**Comment by derekchiang** [comment]
I think this can be closed now since #1075 landed.

**Comment by sjezewski** [comment]
Yup!
